### PR TITLE
Pr/scheduler/api

### PR DESCRIPTION
**Pull request by MethodenMann**

### CODE REVIEW

File: packages/resource-scheduler/src/components/ResourceScheduler/ResourceScheduler.js [hunk 1]
```diff
@@ -1,16 +1,37 @@
...
 import SearchPanel from '../SearchPanel/SearchPanel'
 import SchedulerApp from 'tocco-scheduler/src/main'
+import PropTypes from 'prop-types'
 
 class ResourceScheduler extends React.Component {
+  componentWillMount() {
+    this.props.initialize()
+  }
+
+  paneWidth = 400
```
Comment (ottrob) [inline packages/resource-scheduler/src/components/ResourceScheduler/ResourceScheduler.js]: const paneWidth

File: packages/resource-scheduler/src/components/ResourceScheduler/ResourceScheduler.scss [hunk 2]
```diff
@@ -0,0 +1,14 @@
+.search-panel-wrapper {
+  height: 100%;
+  overflow: auto;
+  padding: 8px;
+}
+
+.Resizer.vertical {
+  background-color: transparent;
+  background-image: url('data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAUAAAAeCAYAAADkftS9AAAAIklEQVQoU2M4c+bMfxAGAgYYmwGrIIiDjrELjpo5aiZeMwF+yNnOs5KSvgAAAABJRU5ErkJggg==');
```
Comment (ottrob) [inline packages/resource-scheduler/src/components/ResourceScheduler/ResourceScheduler.scss]: Why background-image?

File: packages/resource-scheduler/src/components/SearchPanel/SearchPanel.js [hunk 3]
```diff
@@ -1,11 +1,69 @@
...
+  handleSelect = name => selection => {
+    this.props.addCalendarsOfType(name, selection.map(s => s.id))
+  }
+
+  getSearchLists = calendarTypes => {
+    const accordionHeader = (content, name) => {
+      const className = `glyphicon glyphicon-chevron-${this.state.activeKey === name ? 'up' : 'down'}`
+      return <div onClick={() => { this.handleAccordionSelect(name) }}>
+        {content}
+        <i className={className} style={{float: 'right'}}></i>
```
Comment (ottrob) [inline packages/resource-scheduler/src/components/SearchPanel/SearchPanel.js]: shouldn't we put the styling in our sass files (float: right)?

File: packages/resource-scheduler/src/modules/resourceScheduler/reducer.js [hunk 4]
```diff
@@ -0,0 +1,22 @@
+import * as actions from './actions'
+import {reducers} from 'tocco-util'
+
+const addCalendars = (state, {payload}) => ({
+  ...state,
+  calendars: [...state.calendars, ...payload.calendars]
```
Comment (ottrob) [inline packages/resource-scheduler/src/modules/resourceScheduler/reducer.js]: Maybe also make sure duplicates cannot exist?

File: packages/resource-scheduler/src/components/ResourceScheduler/ResourceScheduler.js (hunk 1 above)
Comment (MethodenMann) [reply packages/resource-scheduler/src/components/ResourceScheduler/ResourceScheduler.js]: does not work (its a class)

File: packages/resource-scheduler/src/components/ResourceScheduler/ResourceScheduler.scss (hunk 2 above)
Comment (MethodenMann) [reply packages/resource-scheduler/src/components/ResourceScheduler/ResourceScheduler.scss]: its used to show the dot-image to indicate something resizeable

File: packages/resource-scheduler/src/modules/resourceScheduler/reducer.js (hunk 4 above)
Comment (MethodenMann) [reply packages/resource-scheduler/src/modules/resourceScheduler/reducer.js]: i d like to solve this as soon as we integrate the selectable change

**Pull request merged**